### ISSUE TITLE
Swagger codegen documentation stale link

### ISSUE THREAD

**Issue by lrowen** [f4exb/sdrangel#1755]
At [https://github.com/f4exb/sdrangel/tree/master/swagger/sdrangel#installation](url) a link is given to a fixed version of the swagger qt5 code generator.  This link does not work as of 7/31/2023.  For those who are just starting to get introduced to swagger and how it's integrated in SDRangel, it would be good to have a correct link or correct information here. 

EDIT: If I could just learn to use the github link syntax.... [https://github.com/f4exb/sdrangel/tree/master/swagger/sdrangel#installation](https://github.com/f4exb/sdrangel/tree/master/swagger/sdrangel#installation) is the correct link.... my bad on that one.

**Comment by f4exb** [comment]
Which link?

**Comment by f4exb** [comment]
This is what you should be looking for: https://github.com/f4exb/sdrangel/tree/master/swagger/docker

**Comment by lrowen** [comment]
The link in the page I referenced is [https://github.com/etherealjoy/swagger-codegen/tree/qt5cpp_rework_antis81_patch-1](url) which throws a 404.

Sorry; I just right clicked the link icon in the listing I referenced in the link; the actual file with the link is the README.md in that directory.  The link into the file itself, as given by the github web interface, is [https://github.com/f4exb/sdrangel/blob/master/swagger/sdrangel/README.md#installation](url)

FWIW I'm not using docker.  I understand much of the information is usable outside docker, but I was just giving a 'link no longer available' report for something linked in one of the README.md files.
